.github/workflows/ci_tests.yaml

```diff
@@ -0,0 +1,31 @@
+name: Python package
+
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+    paths-ignore:
+      - 'docs/**'
+      - '**.md'
+
+jobs:
+
+  pytest_tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.9"]
+    container:
+      image: python:${{ matrix.python-version }}
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install flake8 pytest pytest-cov pytest-xdist numpy
```

Comment:
Should we have `/requirements-dev.txt` in addition to `/requirements.txt`?

If we put `flake8`, `pytest`, etc in `/requirements-dev.txt`, it seems that just only CI but also developers can use it.